lib/js-api/test/alfrescoApi.spec.ts

```diff
@@ -84,7 +84,7 @@ describe('Basic configuration test', () => {
             assert.equal(alfrescoApi.config.ticketEcm, mockConfigTicket);
             assert.equal(alfrescoApi.contentClient.config.ticketEcm, mockConfigTicket);
 
-            alfrescoApi.on('ticket_mismatch', () => {
+            alfrescoApi.contentClient.on('ticket_mismatch', () => {
```

Comment:
Why this change? this looks wrong as it's explicitly referring to the contentClient context